### ISSUE TITLE
The unnecessary events are fired when items of a group are changed in the Form

### ISSUE THREAD

**Issue by jsdmitry** [DevExpress/devextreme-vue#330]
I have tested with `vue` the performance is better but the events are fired unnecessary

![dxform-vue](https://user-images.githubusercontent.com/1750751/66770614-78257c00-ee8e-11e9-96a3-80ac7922c7d8.gif)

_Originally posted by @leandrogehlen in https://github.com/DevExpress/DevExtreme/issues/7295#issuecomment-541809440_

**Comment by jsdmitry** [comment]
@leandrogehlen Based on the provided video, it is difficult to determine the cause of the problem. Please attach a simple project illustrating how to reproduce this problem.
Regarding the code example. Would you clarify if it illustrates the problem you described above or some other problem?
```vue
<dx-simple-item
   data-field="fantasia"
   editor-type="dxTextBox"
   v-if="tipoJuridica"
   :label="{text: $t('fantasia')}"
>
export default {
  components: {
    DxForm,
    DxLabel,
    DxSimpleItem,
    DxGroupItem,
    DxRequiredRule,
    DxCustomRule,
    DxEmailRule,
    DxButtonItem
  },

  mixins: [form],

  data() {
    return {
      registro: {
        tipo: 1
      }
    }
  }, 
  computed: {
    tipoJuridica() {
      return this.registro.tipo == 2;
    }
  } 
```

**Comment by leandrogehlen** [comment]
This is my test project

[vue.zip](https://github.com/DevExpress/devextreme-vue/files/3745586/vue.zip)

**Comment by leandrogehlen** [comment]
Some progress about it?

**Comment by geods3** [comment]
it not working... always re-renders the form... and if selectbox has value re-query the datasource of every field....

**Comment by tar-z** [comment]
Hi,
We are working on the issue. Once we have any news, we will update this thread. Thank you for your patience.

**Comment by tar-z** [comment]
The fix will be available in version 19.2.4.
